backend/fms_core/filters.py

```diff
@@ -0,0 +1,23 @@
+import django_filters
```

Comment:
Not sure i would start implementing filters for each viewset ... If we decide to go that way we should do all of them or at least create only the ones that are required because of the batch filtering (samples, container). From what i see here it is also a departure from the pattern we were using before where filters and lookup_expr are liste in the fields. We should go one way or the other not both unless there is a reason.

Reply:
Right, if an entity filter doesnt require a custom filter, I also would not implement a custom filter for that one.